### DISCUSSION TITLE
Custom local fonts for antd 5 theming

### DISCUSSION THREAD

**Discussion by alhawk** [ant-design/ant-design#39346] [Q&A]
Hello, I'm just about to customize the new antd 5 for a react project. Loved it. I previously customize the antd 4 through .less file, but v5 seems having a different approach, I planned to customise the seed token that I see there is a key called `fontFamily` with string value. But where should I import the fonts file since I need to use my local fonts. Do antd provide a way to import the fonts?

**Comment by rooneyhoi** [comment]
as far as I know, it's now not support yet, maybe in the next version

**Reply by chun-wen** [reply]
is there any update info ?

**Comment by spxf** [comment]
one year later, any update? i am facing the same issue!

**Comment by Panchi99** [comment]
I am also facing the same issue. I had used a custom local font but since the v5 upgrade, haven't found a way to inject it. Retaining the ability to use custom local fonts in the new antd v5 seems like a must to me.

**Comment by yomarhuamaniunsch** [comment]
I'm using a webfonts, so I create a global style using antd-style library. (I use woff2, other formats I couldn't test)

```
import {createGlobalStyle} from "antd-style";

const FontApp = createGlobalStyle`

  @font-face {
    font-family: 'Adobe Clean';
    src: url('/fonts/adobe/AdobeClean-Regular.woff2') format('woff2');
    font-weight: normal;
    font-style: normal;
  }
`;

export default FontApp;
```

Then I set the fontFamily in the configProvider 

` <ConfigProvider
      locale={esES}
      theme={{
        token: {
          fontFamily: 'Adobe Clean',
        }
      }}
    >`
